lib/bgsite_official_web/controllers/tag_controller.ex

```diff
@@ -28,7 +29,7 @@ defmodule BgsiteOfficialWeb.TagController do
   end
 
   def show(conn, %{"id" => id}) do
-    tag = Categories.get_tag!(id)
+    tag = Categories.get_tag_2!(id)
```

Comment:
Why?

Reply:
coz i wanna have 2 definitions of `Repo.get_tag` - a default one:
```
  def get_tag!(id) do
    Repo.get!(Tag, id)
    |> Repo.preload(:websites)
  end
```
and one with `order_by` function:
```
def get_tag_2!(id) do
  query =
    from(
      t in Tag,
      where: t.id == ^id,
      select: t,
      preload: [
        :websites,
        websites:
          ^from(
            w in Websites,
            order_by: [desc: w.priority]
          )
      ]
    )
  Repo.one!(query)
end
```

:)

Reply:
However, will change the def name, its inaccurately